google-cloud-http4s/js/src/main/scala/feral/google-cloud/http4s/IOCloudHttpFunction.scala

```diff
@@ -28,18 +28,18 @@ import org.http4s.nodejs.ServerResponse
 import scala.scalajs.js
 import scala.scalajs.js.annotation._
 
-object IOGoogleCloudHttp {
+object IOCloudHttpFunction {
   @js.native
   @JSImport("@google-cloud/functions-framework", "http")
   def http(
```

Comment:
Oh, I missed this one. We can make it private too!
```suggestion
  private[googlecloud] def http(
```